aci/compose.go

```diff
@@ -123,21 +125,29 @@ func (cs aciComposeService) warnKeepVolumeOnDown(ctx context.Context, projectNam
 }
 
 func (cs *aciComposeService) Down(ctx context.Context, projectName string, options compose.DownOptions) error {
-	logrus.Debugf("Down on projectName with name %q", projectName)
-
-	if err := cs.warnKeepVolumeOnDown(ctx, projectName); err != nil {
-		return err
-	}
-
-	cg, err := deleteACIContainerGroup(ctx, cs.ctx, projectName)
-	if err != nil {
-		return err
+	if options.Volumes {
+		return errors.Wrap(errdefs.ErrNotImplemented, "--volumes option is not supported on ACI")
 	}
-	if cg.IsHTTPStatus(http.StatusNoContent) {
-		return errdefs.ErrNotFound
+	if options.Images != "" {
+		return errors.Wrap(errdefs.ErrNotImplemented, "--rmi option is not supported on ACI")
 	}
+	return progress.Run(ctx, func(ctx context.Context) error {
+		logrus.Debugf("Down on projectName with name %q", projectName)
```

Comment:
I think this was a previous renaming gone wrong and it should probably be
```suggestion
		logrus.Debugf("Down on project with name %q", projectName)
```
?